includes/utilities.inc

```diff
@@ -1178,13 +1178,23 @@ function islandora_paged_content_extract_tiff_from_pdf($uri, $offset, $device, $
   $output_file = drupal_tempnam('temporary://', "page{$offset}.tif");
   // Need to link the file because of UTF-8 encoding.
   $copy = islandora_paged_content_create_clean_link($uri, 'pdfforgs');
-  $base_command = '!gs_path -q -dNOPAUSE -dBATCH -sDEVICE=!device -sCompression=lzw -sOutputFile=!output_file -r!resolution, -dFirstPage=!offset -dLastPage=!offset !pdf_uri';
+  // For all arguments see https://ghostscript.com/doc/current/Devices.htm#TIFF.
+  if ($device == 'tiff64nc' || $device == 'tiff48nc') {
+    // Will fail anyway in gs 9.10
+    // Both are 16bits per channel. No compression possible.
+    // Was fixed in:
+    // http://git.ghostscript.com/?p=ghostpdl.git;a=commitdiff;h=f8e77523.
+    $base_command = '!gs_path -q -dNOPAUSE -dBATCH -sDEVICE=!device -sCompression=none -sOutputFile=!output_file -r!resolution -dFirstPage=!offset -dLastPage=!offset !pdf_uri';
```

Comment:
This seems like a bigger problem? Can we not compress Tiffs using GS in versions >= 9.10?

Reply:
We can, but only if using ```tiff32nc``` or ```tiff16nc``` or ```tiffsep```, etc. The issue is that compression actually never ever worked with tiff64nc, it was ignored but it did not fail. 9.10 introduced the issue that running tiff64nc is almost impossible (impossible!) because compression is set to ```lzw``` and 16bits per channel simply don´t compress according to tiff 6.0 specs. So that is actually a bug. Same reason we need to document that. If people need to use tiff64nc (you can get the same results using ```tiffsep``` and using the don't generate separations flag actually)  we need an alternative. One of my previous pulls added actually that alternative to the drop box, but i was afraid i was going to far.
See https://www.ghostscript.com/doc/current/Devices.htm#TIFF 
(https://www.ghostscript.com/doc/9.10/Devices.htm#TIFF is actually the same)
The first ones should generate  "uncompressed output:" and 
-sCompression= is part of tiffsep

In reality, their docs are complex to follow

Reply:
Also GS in versions 9.10 to 9.15 seems? the rest should work. Anyone willing to test every version?

Reply:
No, that seems like overkill. It appears that the fix was first included in the 9.19 release. Should we add that to the note here or just in whatever documentation we do for this?

Reply:
Yeah. Go ahead and add a note(comment) in the main conversation (this ones get obscured by changes) and then we can make sure that goes release as a documentation task.  @whikloj++

Reply:
Perhaps we should remove the mention of "9.10" in the code note? Based on the git commit date and the various release notes it appears that this bug would affect GS versions 9.10 -> 9.18 inclusive.